### DISCUSSION TITLE
[Projects Beta] Export button

### DISCUSSION THREAD

**Discussion by TristanLee9608** [community/community#31983] [Projects and Issues]
Hi, It would be great if the new GitHub projects page had an export button to export excel/csv all issues within, possibly along with all the comments in the issue.
Using RestAPI works, but export button seems more easy and reasonable.

**Comment by Z-K-O** [comment]
Along with all descriptions and custom fields would be greatly appreciated.

**Comment by 2percentsilk** [comment]
@TristanLee9608 @Z-K-O 
Could you tell me a little more about why you want an export of the project? Are you doing manipulations in Excel that we don't currently support? Do you need to transfer these issues somewhere else?

**Reply by Z-K-O** [reply]
> Are you doing manipulations in Excel that we don't currently support?

Yes. I would like an easy way to do math using the data entered in custom fields, first and foremost. Excel seems like the easiest way to do these types of things without asking for a lot of extra GitHub developer time.

It would also happen to give us the benefit of easily creating reports for more general purposes where the recipient would not be required to integrate into our GitHub Organization and learn to navigate it just to observe data that they may only ever need to see once or twice.

**Reply by 2percentsilk** [reply]
Does this newest release start to address the math you're doing for fields?
https://github.blog/changelog/2022-09-27-the-new-github-issues-september-27th-update/#%E2%9E%95-display-sum-of-numeric-fields

It's just the start so we'd love to know what other manipulations you're trying to do!

**Reply by TristanLee9608** [reply]
Newest release mentioned seems very usefull!! Thank you!
In my case, I was considering to use the project board to manage customer support histories. 
When the customer requests their history, it doesn't seem right to just send them the screenshot of the board..hence the excel export button would be appreciated.

**Reply by Z-K-O** [reply]
@2percentsilk It is one step closer. We're collaboratively tracking purchases (from idea inception through the purchasing process and receiving) in this case though, so I need a =SUMPRODUCT() or a manual way of accomplishing the same. And then also per-item optionally applied tax percentages and groupings by Milestone to account for accounts/fiscal years.

Maybe outside of the scope that GitHub wants to support, but it's an ideal platform for us to use for this task given other factors and features. That's why I was thinking easy Excel exports would be helpful.

**Reply by hoeschenbd** [reply]
I would like an export to csv/excel button so that I can prepare a formal estimate for a client or provide completion reports for invoices. We spend time preparing the stories, entering estimates, and tracking status and just need a quick way to get the data exported.

**Comment by Z-K-O** [comment]
@TristanLee9608 What method did you use to get RestAPI exports working? Do you have any critical pointers? I'm not the most experienced in this, since most of my programming is in the hardware sphere, but I do think I could get it working.

**Reply by TristanLee9608** [reply]
@Z-K-O I think I used https://github.com/gavinr/github-csv-tools.

**Reply by Z-K-O** [reply]
Thank you

**Comment by chrisglein** [comment]
Lots of related issues on this. Attempting to help interlink: #5411 #6217 #47733 #48389

**Comment by TristanLee9608** [comment]
I see this feature has been shiped to github.com!!
Thank you!
